applications/serial_lte_modem/src/nrfcloud/slm_at_nrfcloud.c

```diff
@@ -703,8 +705,7 @@ static int handle_at_nrf_cloud_pos(enum at_cmd_type cmd_type,
 
 			/* Parse the MAC address. */
 			len = sizeof(mac_addr_str);
-			err = at_params_string_get(
-				param_list, param_idx, mac_addr_str, &len);
+			err = util_string_get(parser, param_idx, mac_addr_str, &len);
 			if (!err && (len != sizeof(mac_addr_str)
```

Comment:
Increase the space for mac_addr_str to fit the terminating zero and change the check:
```suggestion
			if (!err && (len != strlen(mac_addr_str)
```

Reply:
With old AT-command parser we allowed skipping the null terminating character if it did not fit :facepalm:. I went over the places where `util_string_get` is used, but did not locate any other places where there definitively would not be space for 0.

Reply:
@MirkoCovizzi: Still requires the +1 for buffer size, mentioned above: "Increase the space for mac_addr_str to fit the terminating zero"

Reply:
Updated.